src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/uat/common/TestData.java

```diff
@@ -0,0 +1,12 @@
+package uk.nhs.digital.nhsconnect.lab.results.uat.common;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class TestData {
```

Comment:
Is this needed?

Reply:
No it isn't being used anywhere so I've removed it